tests/python/unittest/test_tir_transform_lower_cross_thread_reduction.py

```diff
@@ -44,7 +47,7 @@ def loop_split(a: T.handle, b: T.handle) -> None:
             with T.block("B"):
                 vi = T.axis.S(128, i)
                 vk = T.axis.R(128, ko * 32 + ki)
-                T.reads([B[vi], A[vi, vk]])
+                T.reads([A[vi, vk]])
```

Comment:
Why are we removing `B[vi]` from `T.reads`? It's actually read, isn't it?

Reply:
I guess starting from some commit, we removed all reduction buffers from `T.reads` 🤔.

One example in other test cases is https://github.com/apache/tvm/blob/9671aee942503815ad2a586406eef11391287ee5/tests/python/unittest/test_tvmscript_roundtrip.py#L2716-L2717

But I can’t remember when it starts from.

Reply:
I see! I just checked myself and found the same thing :-) Thanks for a heads-up!

Reply:
My pleasure!